packages/docs/celo-holder-guide/eth-recovery.md

```diff
@@ -0,0 +1,72 @@
+# Recovering CELO from an Ethereum Wallet
+
+In this guide, steps are outlined to help you recover your [CELO (previously Celo Gold)](../overview.md#background-and-key-concepts) if you accidentally transferred them to an [Ethereum address](https://en.wikipedia.org/wiki/Ethereum#Addresses).
+
+## Prerequisites
+
+This guide assumes that you have access to the following:
+
+- A computer with a [Command Line Interface](https://en.wikipedia.org/wiki/Command-line_interface). You can access it following these instructions:
+    - [Windows](https://www.howtogeek.com/270810/how-to-quickly-launch-a-bash-shell-from-windows-10s-file-explorer/)
+    - [Mac](https://macpaw.com/how-to/use-terminal-on-mac)
+    - [Linux](https://askubuntu.com/questions/196212/how-do-you-open-a-command-line)
+- The [24-word recovery phrase](https://kb.myetherwallet.com/en/security-and-privacy/what-is-a-mnemonic-phrase/) of your Ethereum address
```

Comment:
Should we add a requirement around having the celocli installed?

Reply:
added